pkg/api/api.go

```diff
@@ -111,6 +111,12 @@ func (hs *HTTPServer) registerRoutes() {
 		r.Get("/admin/storage", reqSignedIn, hs.Index)
 		r.Get("/admin/storage/*", reqSignedIn, hs.Index)
 	}
+
+	// feature toggle admin page
+	if hs.Features.IsEnabled(featuremgmt.FlagFeatureToggleAdminPage) {
+		r.Get("/featuremgmt", authorize(ac.EvalPermission(ac.ActionFeatureManagementRead)), hs.Index)
```

Comment:
When I perform a full page refresh on the feature toggles page I still get a 404 from the server

![image](https://github.com/grafana/grafana/assets/46142/f9e1f846-125d-4741-b2ed-000c01377a99)


I think this should be

`r.Get("/admin/featuretoggles", ...`